cypress/support/step-definitions/flat-table.js

```diff
@@ -0,0 +1,168 @@
+import {
```

Comment:
the file name should be changed to `flat-table-steps.js`

Reply:
fixed